sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -272,8 +268,6 @@ def send_options_request(max_retries=5):
         try:
           # This reports channel is READY but connections may fail
           # Seems to be only an issue on Mac with port forwardings
-          if channel:
-            grpc.channel_ready_future(channel).result()
           return job_service.DescribePipelineOptions(
               beam_job_api_pb2.DescribePipelineOptionsRequest())
         except grpc._channel._Rendezvous as e:
```

Comment:
Should make sure this error is caught in the new handler. See comment at new handler.